packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/deploy-cc-from-golang-source-private-data.test.ts

```diff
@@ -0,0 +1,398 @@
+  };
+
+  const pluginOptions: IPluginLedgerConnectorFabricOptions = {
+    instanceId: uuidv4(),
+    dockerBinary: "/usr/local/bin/docker",
+    peerBinary: "/fabric-samples/bin/peer",
+    goBinary: "/usr/local/go/bin/go",
+    pluginRegistry,
+    cliContainerEnv: org1Env,
+    sshConfig,
+    logLevel,
+    connectionProfile,
+    discoveryOptions,
+    eventHandlerOptions: {
+      strategy: DefaultEventHandlerStrategy.NETWORKSCOPEALLFORTX,
+      commitTimeout: 300,
+    },
+  };
+  const connector = new PluginLedgerConnectorFabric(pluginOptions);
+
+  const expressApp = express();
+  expressApp.use(bodyParser.json({ limit: "250mb" }));
+  const server = http.createServer(expressApp);
+  const listenOptions: IListenOptions = {
+    hostname: "localhost",
+    port: 0,
+    server,
+  };
+  const addressInfo = (await Servers.listen(listenOptions)) as AddressInfo;
+  const { port } = addressInfo;
+  test.onFinish(async () => await Servers.shutdown(server));
+
+  await connector.getOrCreateWebServices();
+  await connector.registerWebServices(expressApp);
+
+  const apiUrl = `http://localhost:${port}`;
+
+  const apiClient = new FabricApi({ basePath: apiUrl });
+
+  const contractName = "asset-transfer-private-data";
+
+  const contractRelPath =
+    "../../fixtures/go/asset-transfer-private-data/chaincode-go";
+  const contractDir = path.join(__dirname, contractRelPath);
+
+  const smartContractGoPath = path.join(
+    contractDir,
+    "./chaincode/",
+    "./asset_transfer.go",
+  );
+  const smartContractGoBuf = await fs.readFile(smartContractGoPath);
+  const smartContractGo = {
+    body: smartContractGoBuf.toString("base64"),
+    filepath: "./chaincode/",
+    filename: `asset_transfer.go`,
+  };
+
+  const assetTransferGoPath = path.join(contractDir, "./main.go");
+  const assetTransferGoBuf = await fs.readFile(assetTransferGoPath);
+  const assetTransferGo = {
+    body: assetTransferGoBuf.toString("base64"),
+    filename: `${contractName}.go`,
+  };
+
+  const goModPath = path.join(contractDir, "./go.mod");
+  const goModBuf = await fs.readFile(goModPath);
+  const goMod = {
+    body: goModBuf.toString("base64"),
+    filename: "go.mod",
+  };
+
+  const goSumPath = path.join(contractDir, "./go.sum");
+  const goSumBuf = await fs.readFile(goSumPath);
+  const goSum = {
+    body: goSumBuf.toString("base64"),
+    filename: "go.sum",
+  };
+
+  const privateDataCollectionName = "collections_config.json";
+  const privateDataCollectionsPath = path.join(
+    contractDir,
+    "./" + privateDataCollectionName,
+  );
+  const privateDataCollectionsBuf = await fs.readFile(
+    privateDataCollectionsPath,
+  );
+  const privateDataCollections = {
+    body: privateDataCollectionsBuf.toString("base64"),
+    filename: privateDataCollectionName,
+  };
+
+  const res = await apiClient.deployContractV1({
+    channelId,
+    ccVersion: "1.0.0",
+    sourceFiles: [
```

Comment:
@RafaelAPB If these are the only files needed then I'd remove the other files from the fixtures directory to reduce the size of the PR if possible (trying to keep the git repo lean where possible).
If you have plans to add test coverage for the additional files (such as the java version of the example chain code) then I withdraw my suggestion.

Reply:
Ups - that was a mistake. Those should not be there. Thanks!